### PR TITLE
Add dead letter queue size metric

### PR DESCRIPTION
**Pull request by Deroin**
### What does this PR do?

As discussed in #909 this adds a new metric for dead letter queue size to the logstash integration.

### Motivation

We used this in production before, we're missing it after an infrastructure upgrade and then noticed we never provided it for public use.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

I don't know if I should notify the docs team about this or not. Also don't know if you will manage this checklist above or if I should do it. If there needs to be anything done by me just poke me.

I had to adapt the test config for logstash5 by not only enabling dead letter queue but also creating a new input config which keeps logstash running for a bit. On logstash5 the DLQ is started a few seconds after logstash started and with the stdin input logstash stops before that. I also had to increase the wait time for tests due to that fact. On my local machine it passed now, but the wait time may need to be increased again when it fails on build systems. Unfortunately this depends on the speed of the machine executing tests. For logstash 6 and 7 this is no issue as DQL is started at the same time as the main pipeline.

An alternative would be either to not test for the dead letter queue metric or to find a way to exclude it for logstash5 tests.

### CODE REVIEW

File: logstash/tests/docker/logstash5/logstash.yml [hunk 1]
```diff
@@ -0,0 +1,200 @@
+# Settings file in YAML
```
Comment (coignetp) [inline logstash/tests/docker/logstash5/logstash.yml:1]: I don't think we need all these examples in the config file, let's keep a few lines only

**Review by coignetp — CHANGES_REQUESTED** [review]
Thank you for opening the PR! It looks good to me, I just have one suggestion

Pinging @ervansetiawan as the owner of the check

File: logstash/tests/docker/logstash5/logstash.yml (hunk 1 above)
Comment (ervansetiawan-tr) [reply logstash/tests/docker/logstash5/logstash.yml:1]: That makes sense, @coignetp. Other than that, it looks good.

Comment (Deroin) [reply logstash/tests/docker/logstash5/logstash.yml]: Removed all comment lines

**Review by coignetp — APPROVED** [review]

**Comment by hithwen** [comment]
Note: to release a new version of this check it needs a version bump with `ddev release changelog`

**Pull request merged**